### PR TITLE
Update links in README.md to go to https://langcheck.readthedocs.io

### PR DESCRIPTION
**Pull request by kennysong**
Just a minor README.md change.

### CODE REVIEW

File: docs/notebooks/LangCheck Quickstart.ipynb [hunk 1]
```diff
@@ -1,27 +1,13 @@
 {
```
Comment (liwii) [inline docs/notebooks/LangCheck Quickstart.ipynb:1]: Just curious, what's this quickstart change for? I compared the previous & current notebooks with my eyes but couldn't find any significant difference.

**Review by liwii — APPROVED** [review]
Basically LGTM, just had one question.

File: docs/notebooks/LangCheck Quickstart.ipynb (hunk 1 above)
Comment (kennysong) [reply docs/notebooks/LangCheck Quickstart.ipynb:1]: The text diff is a bit weird, but I deleted the second empty cell in the notebook. You can see the empty cell here: https://github.com/citadel-ai/langcheck/blob/2de006353419b5bef4d76666fa83f57a68fccd45/docs/notebooks/LangCheck%20Quickstart.ipynb

Comment (liwii) [reply docs/notebooks/LangCheck Quickstart.ipynb:1]: Got it, feel free to merge then!

**Pull request merged**